test/OrchardCore.Tests/Data/ContentItemTests.cs

```diff
@@ -45,19 +45,6 @@ public void ShouldSerializeParts()
             Assert.Equal("test", (string)contentItem2.Content.MyPart.Text);
         }
 
-        [Fact]
```

Comment:
This equivalent to

https://github.com/OrchardCMS/OrchardCore/blob/ab07490a2d5ae54180d0af3db27c89287991e567/test/OrchardCore.Tests/Data/ContentItemTests.cs#L61-L72